README.md

```diff
@@ -17,7 +17,7 @@
   - under the License.
   -->
 
-![](site/docs/img/Iceberg-logo.png)
+![](./Iceberg-logo.png)
```

Comment:
how about using this `https://iceberg.apache.org/docs/latest/img/Iceberg-logo.png` instead of adding logo in repo ?